src/helpers/randomArticles.js

```diff
@@ -0,0 +1,16 @@
+
+const getRandomArticles = (count, articles) => {
```

Comment:
Since you are consuming the api now, do you need a function to display random article?

Reply:
☝️